packages/quantic/cypress/integration/case-classification/case-classification.cypress.ts

```diff
@@ -821,4 +829,42 @@ describe('quantic-case-classification', () => {
       });
     });
   });
+
+  describe('when the suggestions are not included in all the options retrieved from Salesforce', () => {
+    it('should display only the select dropdown and log warnings in the console', () => {
+      interceptCaseAssist();
+      mockSfPicklistValues(sfDefaultField, allOptions);
+
+      cy.visit(pageUrl, {
+        onBeforeLoad(win) {
+          stubConsoleWarning(win);
+        },
+      });
+      configure({
+        maxSuggestions: defaultMaxSuggestions,
```

Comment:
if we pass no argument here would it not default to the same thing?

Reply:
you are right! Fixed👌